go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/beevik/ntp v0.3.0
 	github.com/cenkalti/backoff/v4 v4.2.0
 	github.com/containerd/cgroups v1.0.4
-	github.com/containerd/containerd v1.6.10
+	github.com/containerd/containerd v1.6.11
```

Comment:
1.6.12 has the cve fix

Reply:
i think we can update pkgs and then merge this

Reply:
yes, but we don't have it in pkgs yet